wgengine/monitor/monitor.go

```diff
@@ -140,3 +143,42 @@ func (m *Mon) debounce() {
 		}
 	}
 }
+
+// message represents a message returned from an osMon.
+//
+// Currently those are newRouteMessage and newAddrMessage.
+//
+// For historical reasons, a nil value means "some unspecified change
+// you should react to and not ignore".
+type message interface {
+	// Ignore is whether we should ignore this message.
+	ignore() bool
+}
+
+// newRouteMessage is a message for a new route being added.
+type newRouteMessage struct {
+	Src, Dst, Gateway netaddr.IP
+	Table             uint8 // table number ofor OSes that have one
+}
+
+func (m *newRouteMessage) ignore() bool {
+	if runtime.GOOS == "linux" && m.Table == 88 {
+		return true
+	}
+	if cgnatPrefix.Contains(m.Dst) {
+		return true
+	}
+	return false
+}
+
+// newAddrMessage is a message for a new address being added.
+type newAddrMessage struct {
+	Addr  netaddr.IP
+	Label string // netlink Label attribute on Linux (e.g. "tailscale0")
+}
+
+var cgnatPrefix, _ = netaddr.ParseIPPrefix("100.64.0.0/10")
+
+func (m *newAddrMessage) ignore() bool {
+	return cgnatPrefix.Contains(m.Addr)
```

Comment:
Same here, don't ignore the ChromeOS range.

Reply:
Done. (see new tsaddr package)